server/rpm/pbench-server.spec.j2

```diff
@@ -21,9 +21,6 @@ Requires: policycoreutils-python
 Requires: python3-libselinux
 %endif
 
-Requires: npm
```

Comment:
Ouch; good catch. I didn't see this code, but basically the vague suspicion that our dependency on `npm` didn't start with the old dashboard is why I didn't originally remove `npm`. The server's static `web-server` package, which has a `package.json`, depends on a number of `npm` packages to support display of the agent-generated `.html` files.

The generated `<script>` tags should probably refer to an external CDN instead of loading them locally (e.g., `<script src="/static/js/v0.3/d3.min.js"></script>`. We want to get rid of the agent-side generated html, but until then this actually means we can't drop Apache file sharing; even if the UI uses `datasets/inventory` to load the generated HTML into a frame, that code will still reference the Javascript packages from the server, and they're outside the dataset tarball.

So, yeah; big wrench, expertly lobbed right in the center of the works. We need our `npm` back, until the generated agent html is fixed or removed... which isn't going to happen for 0.69 or 0.71.

Reply:
> So, yeah; big wrench, expertly lobbed right in the center of the works. We need our `npm` back, until the generated agent html is fixed or removed... which isn't going to happen for 0.69 or 0.71.

Oh yeah; and if we had a CI pipeline that built and deployed an RPM, we'd have known this by now. 👍🏻

Reply:
Yeah...about that...Peter just talked me out of including the Dashboard in the Server RPM, so the pipeline may have gotten a little longer....